### PR TITLE
Add SSH keys and cPanel logs

### PR DESCRIPTION
**Pull request by Zawadidone**

### CODE REVIEW

File: acquire/acquire.py [hunk 1]
```diff
@@ -1326,12 +1335,20 @@ class Home(Module):
     SPEC = [
         ("glob", "/root/.*[akz]sh*"),
         ("dir", "/root/.config"),
+        ("dir", "/root/.ssh"),
         ("glob", "/home/*/.*[akz]sh*"),
-        ("glob", "/home/*/.config"),
+        ("glob", "/home/*/.config/*"),
+        ("glob", "/home/*/.ssh/*"),
```
[the remark this answers is not in the document]
Comment (Zawadidone) [reply acquire/acquire.py]: Added patterns for the paths

File: acquire/acquire.py [hunk 2]
```diff
@@ -1323,22 +1332,43 @@ class Boot(Module):
...
+            "/root/.ssh/*.pub",
+            "/root/.ssh/authorized_keys*",
+            "/home/*/.ssh/*.pub",
+            "/home/*/.ssh/authorized_keys*",
+            "/Users/*/.ssh/*.pub",
+            "/Users/*/.ssh/authorized_keys*",
+        ]
+        for pattern in patterns:
+            for path in target.fs.glob(pattern):
+                collector.collect_file(path, outpath=path)
```
[the remark this answers is not in the document]
Comment (Schamper) [reply acquire/acquire.py:1370]: Might as well add `from_user_home` is you're going to do this.

[the remark this answers is not in the document]
Comment (Zawadidone) [reply acquire/acquire.py:1370]: `from_user_home` is not supported on OS-X (https://github.com/fox-it/dissect.target/issues/300)

[the remark this answers is not in the document]
Comment (Zawadidone) [reply acquire/acquire.py]: Thanks for the suggestion. The module is also added to ESXi (https://kb.vmware.com/s/article/1020530).

**Comment by Zawadidone** [comment]
I guess you mean two PR's?

**Comment by Zawadidone** [comment]
https://github.com/fox-it/acquire/pull/76
https://github.com/fox-it/acquire/pull/77

**Pull request closed without merging**